community/cacti/APKBUILD

```diff
@@ -50,15 +54,15 @@ package() {
 	# move variable and writable data to cacti's home
 	local dir; for dir in cache plugins resource rra scripts; do
 		mv "$pkgdir"/usr/share/webapps/$pkgname/$dir \
-			"$pkgdir"/var/lib/$pkgname/$dir || return 1
-		chown -R $pkgusers:$pkggroups "$pkgdir"/var/lib/$pkgname/$dir
+			"$pkgdir"/var/lib/$pkgname/$dir
+		chown -R $pkgname: "$pkgdir"/var/lib/$pkgname/$dir
 		ln -s /var/lib/$pkgname/$dir "$pkgdir"/usr/share/webapps/$pkgname/$dir
 	done
 
 	# move log to /var/log
 	mv "$pkgdir"/usr/share/webapps/$pkgname/log \
-		"$pkgdir"/var/log/$pkgname || return 1
-	chown -R $pkgusers:$pkggroups "$pkgdir"/var/log/$pkgname
+		"$pkgdir"/var/log/$pkgname
+	chown -R $pkgname: "$pkgdir"/var/log/$pkgname
```

Comment:
Are you sure web server does not need to have write permissions to log directory?